src/main/java/com/checkout/reconciliation/PaymentReportData.java

```diff
@@ -41,10 +40,10 @@ public final class PaymentReportData extends Resource {
     private String cardCategory;
 
     @SerializedName("issuer_country")
-    private CountryCode issuerCountry;
```

Comment:
I found this issue on .NET, running some Reconciliation tests with the following error
`Newtonsoft.Json.JsonSerializationException: Error converting value "GBR" to type 'System.Nullable`1[Checkout.Common.CountryCode]'. Path 'data[3].merch.`

A deeper investigation after seeing the response payload, the `merchant_country` contains a three letter country, and in code we mapped as two letter field, so I made the assumption that this could happen in production as well.
`"id": "pay_rjl7w7usvrq2lo3cow24rpnvry",
            "processing_currency": "GBP",
            "payout_currency": "AED",
            "requested_on": "2022-05-20T11:33:45.174",
            "channel_name": "Channel API V2",
            "payment_method": "VISA",
            "card_type": "DEBIT",
            "card_category": "Consumer",
            "issuer_country": "GB",
            "merchant_country": "GBR",
            "region": "Domestic",
            "actions": [`

Another thing that support this idea, in the documentation these values are string, and they are not specified as Two letters like any other country, so I believe the API supports different values, rather than two letters code
![image](https://user-images.githubusercontent.com/30295119/171872151-af9ece2c-212a-4da5-a5dd-a65d36463bbc.png)

Reply:
It would be important to confirm with the team if it makes sense to support 3 digit country codes (ISO3). [From my understanding](https://www.checkout.com/docs/four/resources/codes/country-codes), most endpoints return ISO2 codes. If there are endpoints returning ISO3 I think that the long term solution is to adapt the `CountryCode` to support 3 digits.

```    
@SerializedName(value = "GB", alternate = "GBR")
GB("44"),
```